owasp-top10-2017-apps/a5/ecommerce-api/app/handlers/handlers.go

```diff
@@ -13,14 +15,44 @@ func HealthCheck(c echo.Context) error {
 	return c.String(http.StatusOK, "WORKING\n")
 }
 
+func extractClaims(tokenStr string) (jwt.MapClaims, bool) {
+	hmacSecretString := "secret" // Value
```

Comment:
Ops, is that a hardcoded credential on your code? 🧐

Reply:
Where???? I can't see it =D

<sub><sub><sub>Fixed!</sub></sub></sub>

Reply:
🤣 👏🏻